ste/mgr-JobPartTransferMgr.go

```diff
@@ -89,7 +89,7 @@ type IJobPartTransferMgr interface {
 }
 
 type TransferInfo struct {
-	BlockSize              uint32
+	BlockSize              int64
```

Comment:
>int64 [](start = 24, length = 5)

This is an upgrade to the plan file version, please make sure to increment the corresponding constant

Reply:
Needs an update to plan version